extensions/wikia/ArticleVideoMVP/ArticleVideoMVP.hooks.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class ArticleVideoMVPHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		global $wgVideoMVPArticles, $wgCityId;
+
+		$wg = F::app()->wg;
+		$articleId = $wg->Title->getArticleID();
+
+		if ( isset( $wgVideoMVPArticles[$wgCityId] ) &&
+		     isset( $wgVideoMVPArticles[$wgCityId][$articleId] )
+		) {
+			\Wikia::addAssetsToOutput( 'premium_mvp_scss' );
+			\Wikia::addAssetsToOutput( 'premium_mvp_js' );
+		}
+
+		return true;
+	}
+
+	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $out ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$out". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)